### PR TITLE
Update installation instructions for logs, metrics and traces

### LINKED ISSUE #657: installing-logging-metrics-traces.md is stale
## Expected Behavior
The command line to install monitoring components still works.

## Actual Behavior
As we have done some refactoring, the below command no longer works.
```shell
kubectl apply --recursive --filename config/monitoring/100-common \
   --filename config/monitoring/150-elasticsearch \
   --filename third_party/config/monitoring/common \
   --filename third_party/config/monitoring/elasticsearch \
   --filename config/monitoring/200-common \
   --filename config/monitoring/200-common/100-istio.yaml
```

### PR DESCRIPTION
**Pull request by mdemirhan**
Fixes #657

Original PR is at: https://github.com/knative/docs/pull/613 - I closed the original one due to conflicts and updated that with 0.3.0 changes.

### CODE REVIEW

**Comment by mdemirhan** [comment]
Original PR is at: https://github.com/knative/docs/pull/613 - I closed the original one due to conflicts and updated that with 0.3.0 changes.

**Comment by mdemirhan** [comment]
@samodell The build test failed with dead links, though it is something I want to put there as an example. Anyway that we can skip that error?

**Comment by samodell** [comment]
This is a known issue (#750), and I don't believe there is a way to override it, but @adrcunha  is working on a fix.

File: serving/installing-logging-metrics-traces.md [hunk 1]
```diff
@@ -1,111 +1,109 @@
...
+   prometheus-system-1                   1/1       Running   0          2d
+   ```
+
+   Tip: Hit CTRL+C to exit watch mode.
+
+[Accessing Metrics](./accessing-metrics.md) for more information about metrics in Knative.
+
+## Logs
+
+Knative offers three different setups for collecting logs:
```
Comment (samodell) [inline serving/installing-logging-metrics-traces.md]:
```suggestion
Knative offers three different setups for collecting logs. Choose one to install:
```

File: serving/installing-logging-metrics-traces.md [hunk 2]
```diff
@@ -124,85 +122,101 @@ application logs and `Zipkin` for request traces.
...
+    git clone https://github.com/knative/serving knative-serving
+    cd knative-serving
+    git checkout v0.3.0
+    ```
 
 1. Choose a container image that meets the
-   [Fluentd image requirements](fluentd/README.md#requirements). For example,
-   you can use a public image. Or you can create a custom one and upload the
-   image to a container registry which your cluster has read access to.
+   [Fluentd image requirements](fluentd/README.md#requirements). For example, you can use a
```
Comment (samodell) [inline serving/installing-logging-metrics-traces.md]: Should we provide a link to an appropriate image?

**Pull request merged**